.github/workflows/deploy-dev.yml

```diff
@@ -119,3 +120,19 @@ jobs:
             --set image.tag=$IMAGE_TAG
             --set ckan.psql.initialize=false
             --set solr.initialize.enabled=false
+  notify:
+    runs-on: ubuntu-latest
+    if: ${{ always() }}
```

Comment:
Always for dev might get loud, though I guess let's start loud and tune down if we feel swamped 🙉